lib/ebpf/sample.c

```diff
@@ -11,3 +11,6 @@ int xdp_pass(struct xdp_md *ctx)
     return XDP_PASS;
 }
 
+__section("license")
+char _license[] = "GPL";
```

Comment:
I'm not sure we can claim that sample.c is GPL given the license of the project. Clearly this is necessary but we may need to iron out some legal details before this section can be included.

Reply:
Sure. For now, we can go with it I think.